esx_service/vmdk_ops.py

```diff
@@ -656,11 +656,11 @@ def attachVMDK(vmdk_path, vm_uuid):
 
 
 # Return error, or None for OK.
-def detachVMDK(vmdk_path, vm_uuid):
+def detachVMDK(full_vol_name, vm_uuid):
```

Comment:
so is it  a name (vol@DS) or a path? 
A comment (in a dockstring) would be good